TRLevelControl/IO/TRLevelReader.cs

```diff
@@ -1,7 +1,97 @@
-﻿namespace TRLevelControl;
+﻿using ICSharpCode.SharpZipLib.Zip.Compression.Streams;
+using TRLevelControl.Model;
+
+namespace TRLevelControl;
 
 public class TRLevelReader : BinaryReader
 {
     public TRLevelReader(Stream stream)
         : base(stream) { }
+
+    public TRLevelReader Inflate(TR4Chunk chunk)
+    {
+        chunk.UncompressedSize = ReadUInt32();
+        chunk.CompressedSize = ReadUInt32();
+
+        chunk.CompressedChunk = new byte[chunk.CompressedSize];
+        for (uint i = 0; i < chunk.CompressedSize; i++)
+        {
+            chunk.CompressedChunk[i] = ReadByte();
+        }
```

Comment:
```
chunk.CompressedChunk = ReadBytes(chunk.CompressedSize)
```

Reply:
Had to cast to int, and OG tests are fine with this. I suppose we could make an override for `ReadBytes(long count)` if preferable?

![image](https://github.com/LostArtefacts/TR-Rando/assets/33758420/8240e311-1fda-4b5a-8978-350b15c178b9)